.env.template

```diff
@@ -1,5 +1,6 @@
 API_URL=http://localhost:8003
 BLOCK_EXPLORER_URL=http://localhost:3000
+INCENTIVIZED_TESTNET_URL=http://localhost:3001
```

Comment:
Is the testnet url still in use? It might be only for old snapshot data

Reply:
Yeah if its not used anywhere, let's remove the variable.

Reply:
https://github.com/iron-fish/ironfish-api/pull/1536